anyway/parsers/news_flash_db_adapter.py

```diff
@@ -24,24 +24,33 @@ def execute(self, *args, **kwargs):
     def commit(self, *args, **kwargs):
         return self.db.session.commit(*args, **kwargs)
 
+    def recreate_table_for_location_extraction(self):
+        with self.db.session.begin():
+            self.db.session.execute("""TRUNCATE cbs_locations""")
+            self.db.session.execute("""INSERT INTO cbs_locations
```

Comment:
What happens when user code that uses this table runs in parallel (from a different process) to this code?